docs/user_guide/capsules/isolation.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: documentation
+title: Capsule Isolation
+sidebar: sidebars/documentation.html
+---
+
+# Capsule Isolation
+
+The goal of Capsule Isolation is to provide a single endpoint for all of a client's communication, so that in remote network segments, you need only open Firewall ports to the Capsule itself.
+
+![Diagram of Communication](./isolation.png)
+
+## Content and Configuration Services
+
+There are 4 primary areas that require client communication:
+
+### Package Installation
+
+That is, yum.  Katello Capsules by default have the Pulp feature, which mirrors content for the selected Lifecycle Environments.  Enable the Content Mirror with `--pulp=true` on the capsule-installer. Clients need to be able to communicate with the Capsule on port 443/tcp.
```

Comment:
I'd drop the first sentence. Should `Content Mirror` be all caps? is that a proper thing? What about a common formatting for each section that goes:

```
stuff. stuff. thjings.

Firewall Ports: 443/tcp
```